ganga/GangaCore/Utility/Shell.py

```diff
@@ -117,13 +117,11 @@ def __init__(self, setup=None, setup_args=[]):
         """
 
         if setup is not None:
-            self.env = dict(os.environ)
+            self.env = os.environ
             execute('source {0} {1}'.format(setup, " ".join(setup_args)), shell=True, env=self.env, update_env=True)
 
         else:
-            # bug #44334: Ganga/Utility/Shell.py does not save environ
```

Comment:
I assume that the `bug #44334` is a reference to something very old that we don't care about.